magento/adobe-stock-integration
# [Refactoring] Model view responsibility separation

### Description (*)
Model view responsibility separation

### Manual testing scenarios (*)
1. Check Test connection button in the system configuration
2. Check that proper error is displayed when performing the following operation while the API Key is incorrect:
 - open Adobe Stock grid
 - save preview image
 - license image
 - saving licensed image
 - Open preview for adobe stock image that contains related images
